<?xml version="1.0" encoding="utf-8"?>
<launch>

  <!-- Load the URDF into the ROS Parameter Server -->
  <param name="urdf"
	 command="cat '$(find baxter_description)/urdf/baxter.urdf'" />
  <param name="root_name" value="torso" />
  <param name="grav_right_name" value="right_wrist" />
  <param name="grav_left_name" value="left_wrist" />
  <node name="baxter_kinematics_left" pkg="baxter_kinematics" type="kinematics" respawn="false" output="screen" args="left">
</node>
  <node name="baxter_kinematics_right" pkg="baxter_kinematics" type="kinematics" respawn="false" output="screen" args="right">
  </node>


</launch>
